images/pulp_ci_centos/Containerfile

```diff
@@ -31,7 +31,20 @@ RUN dnf -y module enable postgresql:13 && \
     dnf -y install redis && \
     dnf clean all
 
-RUN pip install opentelemetry-sdk opentelemetry-instrumentation-wsgi opentelemetry-distro[otlp]
+RUN pip install \
+opentelemetry-sdk \
+opentelemetry-instrumentation \
+opentelemetry-instrumentation-dbapi \
+opentelemetry-instrumentation-logging \
+opentelemetry-instrumentation-urllib \
+opentelemetry-instrumentation-wsgi \
+opentelemetry-instrumentation-aiohttp-client \
+opentelemetry-instrumentation-botocore \
+opentelemetry-instrumentation-django \
+opentelemetry-instrumentation-psycopg2 \
+opentelemetry-instrumentation-redis \
+opentelemetry-instrumentation-requests \
+opentelemetry-distro[otlp]
```

Comment:
I was not so sure if we use `requests` or not, so I left it.
Make sense to remove it?

Also, I left `logging` since it seems good to obtain spans info from the logs itself, at least in this experimental stage we're working on.